cmd/mt-store-cat/main.go

```diff
@@ -247,12 +256,15 @@ func main() {
 
 	fmt.Printf("# Keyspace %q:\n", *cassandraKeyspace)
 
+	span := tracer.StartSpan("mt-store-cat " + format)
```

Comment:
isn't the only purpose of initializing the tracer just to prevent stuff from calling methods on a `nil` value? why bother starting a span then?

Reply:
the span needs to be non-nil otherwise we get a panic.
plus also i envision at some point we'll actually start collecting traces from this tool.